demo/respeaker-rpi0/porcupine_demo_mic.c

```diff
@@ -9,14 +9,14 @@
     specific language governing permissions and limitations under the License.
 */
 
+#include <alsa/asoundlib.h>
+#include <asm/ioctl.h>
 #include <dlfcn.h>
 #include <errno.h>
+#include <linux/spi/spidev.h>
 #include <signal.h>
 #include <stdio.h>
 #include <stdlib.h>
```

Comment:
great! now if you put a new line between `#include <stdlib.h>` and `#include <alsa/asoundlib.h>` here, they are considered two separate blocks